MIGRATION-V6.md

```diff
@@ -0,0 +1,146 @@
+# Upgrading to ibm-watson@6.0
+
+_Note: If migrating from a version less than 5.0, also see the [v5 migration guide wiki](https://github.com/watson-developer-cloud/node-sdk/wiki/v5-Migration-Guide)._
+
+- [Breaking Changes](#breaking-changes)
+  - [Support for callbacks dropped](#support-for-callbacks-dropped)
+  - [Support for Node v10 dropped](#support-for-node-v10-dropped)
+  - [Breaking changes by service](#breaking-changes-by-service)
+
+- [New Features by Service](#new-features-by-service)
+
+## Breaking changes
+
+#### Promises
+The detailed response is always returned. The `return_response` parameter is removed.
+```js
+const response = await listWorkspaces();
+console.log(response.result); // prints the body
+console.log(response.headers); // prints the headers
+```
+
+#### Support for Callbacks Dropped
+Support for callbacks has been dropped in favor of promises. All functions will no longer accept a callback parameter. An example of replacing a callback with a promise using `await` is shown below
+```js
+//Callback
+const assistant = new AssistantV2(options);
+assistant.message(params, (err, res) => {
+  if(err){
+    console.log(err) //Print out error
+    return
+  }
+  console.log(res.result) //Print out body
+});
+```
+```js
+//Promise
+const assistant = new AssistantV2(options);
+try{
+  await res = assistant.message(params);
+  console.log(res.result) //Print out body
+} catch(err){
+  console.log(err) //Print out error
+}
+
+```
+
+### Support for Node v10 Dropped
+The SDK no longer supports Node version 10, as reflected in the `engines` property in the package.json file. Version 6 reached end of life in April 2019 and Version 8 reaches end of life on 31 December 2019.
```

Comment:
We're only dropping v10 so there's no need to document when v6 and v8 ended. Replace that with the end date of v10